Additional functions with new nsl-assembler https://github.com/Frisch12/nsl-assembler

Frisch12 opened the pull request.

idleberg commented:
Oh wow, there are actually people using this package!

Never heard of `@variable`, what are they used for? You should probably rename it, since there are currently two entries called `variable.name.nsl`. Will merge the changes afterwards.

Thank you!

Frisch12 commented:
@$variable is used for variables defined in some nsis scopes (like custom dialogs, builded with some thrid party tools). nsL-assembler from sourceforge throw an error if you use them in nsl. With my changes (see https://github.com/Frisch12/nsl-assembler) i've fixed this "bug". 
Furthermore I've fixed some other bugs in nsL-assembler, since the project on sourceforge is not maintained anymore.

idleberg commented:
Ah, I see. I think AfrowUK, the author of nsL Assembler, is still fairly active on the [forum](http://forums.winamp.com/forumdisplay.php?forumid=65). Maybe that helps to get your changes merged into the official version.
